Cargo.toml

```diff
@@ -27,3 +27,7 @@ panic = "abort"
 [profile.release]
 panic = "abort"
 lto = true
+strip = "none"
```

Comment:
Just leaving a note here that we probably don't want to merge it just yet as the optional tests (benches) are failing, because they now use unstripped binaries and take longer. We're still looking into that.

Reply:
They also fail on https://github.com/firecracker-microvm/firecracker/pull/4798 for example, which only touched python code, so I don't think it has anything to do with stripping (in fact, `cargo bench` doesn't use the `release` profile: https://doc.rust-lang.org/cargo/commands/cargo-bench.html#description). The new queue benchmarks simply seem very volatile

Reply:
Indeed, it looks like that.
Shall we add an entry in the changelog for that?

Reply:
Added changelog entry.